continuous_integration/install.sh

```diff
@@ -76,3 +76,4 @@ if [ -z "$NUMPY_VERSION" ]; then
 fi
 
 python setup.py install
+pip install nose-exclude
```

Comment:
I think the fastest "fix" is to strip the doctests. This seems to work for me, not sure how to do the same with sed and not perl:

`find dill -name "*.py" | xargs perl -pi -e 's@^\s*(>>>|\.\.\.).+\n@@'`